Client/Ecosia/AppDelegate/Leaves/TelemetryAppDelegate.swift

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+
+final class TelemetryAppDelegate: AppDelegateLeaf {
+    
+    private var profile: Profile
```

Comment:
Same here with `profile`, can't it be removed?

Reply:
`AppLaunchUtil.swift`